codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/customize/RequiredCustomizations.kt

```diff
@@ -58,6 +60,8 @@ class RequiredCustomizations : ClientCodegenDecorator {
         // Add rt-tokio feature for `ByteStream::from_path`
         rustCrate.mergeFeature(Feature("rt-tokio", true, listOf("aws-smithy-http/rt-tokio")))
 
+        rustCrate.mergeFeature(TestUtilFeature)
```

Comment:
Checking I understand this:

We have to manually add this feature here so that it gets added to the generated `Cargo.toml`. This is because we don't have magical support for when stuff is feature-gated _(as opposed to how we add deps to a `Cargo.toml`.)_